x/crosschain/keeper/initiate_outbound_test.go

```diff
@@ -427,10 +439,52 @@ func TestKeeper_ProcessInboundProcessCrosschainMsgPassing(t *testing.T) {
 		observerMock.On("GetChainNonces", mock.Anything, receiverChain.ChainName.String()).
 			Return(observertypes.ChainNonces{}, false)
 
-		// call ProcessInbound
+		// call InitiateOutbound
 		cctx := GetERC20Cctx(t, receiver, *receiverChain, "", amount)
-		k.ProcessInbound(ctx, cctx)
+		err := k.InitiateOutbound(ctx, cctx)
+		require.Nil(t, err)
 		require.Equal(t, types.CctxStatus_Aborted, cctx.CctxStatus.Status)
 		require.Contains(t, cctx.CctxStatus.StatusMessage, "cannot find receiver chain nonce")
 	})
 }
+
+func TestKeeper_InitiateOutboundFailures(t *testing.T) {
+	t.Run("should fail if chain info can not be found for receiver chain id", func(t *testing.T) {
+		k, ctx, _, _ := keepertest.CrosschainKeeperWithMocks(t, keepertest.CrosschainMockOptions{
+			UseFungibleMock: true,
+			UseObserverMock: true,
+		})
+
+		// Setup mock data
+		receiver := sample.EthAddress()
+		amount := big.NewInt(42)
+		receiverChain := getValidEthChain()
+		receiverChain.ChainId = 123
+		// call InitiateOutbound
+		cctx := GetERC20Cctx(t, receiver, *receiverChain, "", amount)
+		err := k.InitiateOutbound(ctx, cctx)
+		require.NotNil(t, err)
+		require.Contains(t, err.Error(), "chain info not found")
```

Comment:
```suggestion
		require.ErrorContains(t, err, "chain info not found")
```
Last comment for assertion.

Would be better to use:
```
NoError
Error
ErrorContains
ErrorIs
```